client/blocks/reader-full-post/_content.scss

```diff
@@ -1,3 +1,6 @@
+@import '@wordpress/base-styles/variables';
```

Comment:
@mehmoodak Do you know why this would cause the following Docker build error?

<img width="597" alt="Screenshot 2025-03-27 at 18 27 34" src="https://github.com/user-attachments/assets/388adeed-c0fb-4456-8fa1-152786af1c69" />